services/frontend-service/src/ui/Pages/Home/Home.tsx

```diff
@@ -26,8 +33,18 @@ export const Home: React.FC = () => {
     const searchedApp = useSearchedApplications(filteredApps, appNameParam);
 
     const apps = Object.values(searchedApp);
+
+    const { app, version } = useReleaseDialog(({ app, version }) => ({ app, version }));
+
+    const envs = useCurrentlyDeployedAt(app, version);
+    let releaseInfo = useReleaseInfo(app, version);
+
+    if (releaseInfo === undefined) {
+        releaseInfo = {};
+    }
     return (
         <main className="main-content">
+            <ReleaseDialog app={app} version={version} release={releaseInfo} envs={envs}></ReleaseDialog>
```

Comment:
```suggestion
            <ReleaseDialog app={app} version={version} release={releaseInfo} envs={envs}/>
```
if there are no children, please use one closed tag only